docs/source/theory.rst

```diff
@@ -272,9 +272,76 @@ Finally, convective heat fluxes can be applied to boundaries:
 
 where :math:`h` is the heat transfer coefficient and :math:`T_{\mathrm{ext}}` is the external temperature.
 
----------------
+Kinetic surface model
+^^^^^^^^^^^^^^^^^^^^^
+
+Modelling hydrogen retention or outgassing might require considering the kinetics of surface processes. 
+A representative example is the hydrogen uptake from a gas phase, when the energy of incident atoms/molecules is not high enough to 
+overcome the surface barrier for implantation. The general approach to account for surface kinetics :cite:`Pick1985, Hodille2017, Guterl2019, Schmid2021` consists in 
+introducing hydrogen surface species :math:`c_\mathrm{s}`. 
+
+Evolution of hydrogen surface concentration is determined by the atomic flux balance at the surface, as sketched in the simplified energy diagram below.
+
+.. figure:: images/potential_diagram.png
+    :align: center
+    :width: 800
+    :alt: Idealised potential energy diagram for hydrogen near a surface of an endothermic metal. Energy levels are measured from the :math:`\mathrm{H}_2` state
+
+    Idealised potential energy diagram for hydrogen near a surface of an endothermic metal. Energy levels are measured from the :math:`\mathrm{H}_2` state
+
+The governing equation for surface species is:
+
+.. math::
+    :label: eq_surf_conc
+    
+    \dfrac{d c_\mathrm{s}}{d t} = J_\mathrm{bs} - J_\mathrm{sb} + J_\mathrm{vs}
+
+where :math:`J_\mathrm{bs}` is the flux of hydrogen atoms from the subsurface (bulk region just beneath the surface) onto the surface, 
+:math:`J_\mathrm{sb}` is the flux of hydrogen atoms from the surface into the subsurface, and :math:`J_\mathrm{vs}` is the net flux of hydrogen 
+atoms from the vacuum onto the surface. The current model does not account for possible surface diffusion and, therefore, is limited to
+one-dimensional hydrogen transport simulations.
```

Comment:
Well in theory, you could use this model in 2D or 3D if the diffusion is zero. For example if you have an homogenous J_vs on the surface ?

At the moment it's a technical limitation. The steady state version of it #800 could be applied to multi dimension problems

Reply:
In theory, yes. However, I think it can't be used if there is any inhomogeneity (e.g., temperature distribution over the surface, solute source distribution). 

Overall, I wanted to highlight that currently it's 1D+0D. Maybe, we can keep only the part about: The current model does not account for possible surface diffusion?

Reply:
You are probably right. That's of course if surface diffusion is faster than the other processes.

We want to highlithg that it's only available for 1D problems in FESTIM. We also want to keep that the current model doesn't account for potential surface diffusion 👍

Reply:
> We want to highlithg that it's only available for 1D problems in FESTIM.

Shall we add a `warning`? Though we have warnings in API.